### PR TITLE
Creating InlineSidebar and Section in Layout

### PR DESCRIPTION
**Pull request by DavidHollins6**
### 💬 Description
There are two biggie changes in this PR:

There is now a `InlineSidebar`, which is a sidebar that is not positioned absolute on top of the page, instead it is a part of the flow!

To help with this, there's another addition `Section`. This is just a layout component to group other layout components together. In this case, we have grouped the `SubHeader`, `Body` and `Footer` together in a `Section`, so the sidebar can be next to them!

### 🚪 Start Points
`stories/components/Layout.js` - You'll find examples of these new components in the story!

### ✅ Checklist
- [ ] Tests written
- [ ] Browser tested
- [ ] Added to documentation
- [ ] Added to storybook

### CODE REVIEW

File: lib/Layout/Body.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { node, string } from 'prop-types';
 export function Body({ children, className }) {
   return (
     <div
-      className={`ql-scroll-container relative py-10 w-full flex flex-col flex-1 min-h-0 overflow-y-auto container ${className}`}
+      className={`ql-scroll-container relative w-full flex flex-col flex-1 min-h-0 overflow-y-auto container ${className}`}
```
Comment (DavidHollins6) [inline lib/Layout/Body.js:7]: I removed this as a small change in this PR because the body should not choose padding!

**Comment by DavidHollins6** [comment]
[Refactor revision history to use new Layout](https://trello.com/c/MsnDBDsf/395-refactor-revision-history-to-use-new-layout)

File: lib/Layout/InlineSidebar.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { node, string } from 'prop-types';
+
+export function InlineSidebar({ children, className }) {
+  return <div className={`${className}`}>{children}</div>;
```
Comment (altaudio) [inline lib/Layout/InlineSidebar.js:5]: 🚩 Can we make this an `aside` tag to make it more semantic?

File: lib/Layout/Section.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { node, string } from 'prop-types';
+
+export function Section({ children, className }) {
+  return <section className={`${className}`}>{children}</section>;
```
Comment (altaudio) [inline lib/Layout/Section.js:5]: 💚 Could also be:
```suggestion
  return <section className={className}>{children}</section>;
```

**Pull request merged**